engines/bops_config/spec/system/application_types_spec.rb

```diff
@@ -104,7 +127,7 @@
     visit "/application_types/#{application_type.id}"
     expect(page).to have_selector("h1", text: "Review the application type")
 
-    within "dl div:nth-child(3)" do
+    within "dl div:nth-child(4)" do
```

Comment:
Not for this PR but it might be worth reviewing some of these tests that reference a particular child element by number — I had to alter one recently too. Perhaps we could give list entries etc an ID and then be able to use `within` on that.

Reply:
Yeah that is usually my preference using the id but i've gone with being consistent with how the other tests are set up